runhouse/resources/hardware/cluster.py

```diff
@@ -918,6 +924,17 @@ def restart_server(
                 )
                 cluster_cert_path = f"{base_caddy_dir}/{self.cert_config.CERT_NAME}"
 
+        if enable_observability is None:
+            enable_observability = rh.configs.observability_enabled
+
+            # edge case: user disabled cluster observability and then wants to enable it again.
+            if not enable_observability:
+                rh.configs.enable_observability()
```

Comment:
do we want to be enabling or disabling at the config level? we should prob allow the user to control this at the cluster level, rather than globally